modules/scalacheck-toolbox-datetime/src/main/scala/com/fortysevendeg/scalacheck/datetime/YearRange.scala

```diff
@@ -0,0 +1,27 @@
+package com.fortysevendeg.scalacheck.datetime
+
+trait YearRange {
+  val min: Int
+  val max: Int
+}
+
+object YearRange {
+
+  /** The default range of years. Set to a safe subset shared by both `java.time` and `org.joda.time`. */
+  implicit def default: YearRange = new YearRange {
+    val min: Int = -292275054
+    val max: Int = 292278993
+  }
+
+  /** Defines a year range between 1970 and the max year you specify. */
+  def epochTo(year: Int) = new YearRange {
+    override val min = 1970
+    val max          = year
```

Comment:
nit: why is one marked with `override` but not the other? Same goes for the method below.

Reply:
An earlier version had min pre-set as `-max`, and I just forgot to remove the override keyword. My bad. Fixed.